metadata.json

```diff
@@ -92,12 +92,6 @@
         "15"
       ]
     },
-//    {
-//      "operatingsystem": "OpenSuSE",
-//      "operatingsystemrelease": [
-//        "13.1"
-//      ]
-//    },
```

Comment:
`pdk update ` did remove this deactive section.
Don't know why it got deactivated in first place.

Reply:
When I ran pdk test unit I got errors than nsswitch module doesn't support openSuse. You don't get those?

Reply:
I didn't try. It shouldn't complain when `manage_nsswitch` is set to false for OpenSuse. But to speak frankly I don't know if that would make the problem even worse ;)

Reply:
Seems that OpenSuse 13.x is out of support anyway.
https://en.opensuse.org/Lifetime#Discontinued_distributions

Reply:
Added a comment about it here https://github.com/ghoneycutt/puppet-module-pam/pull/243/files/16cbb9d3b04e8951920e798eab80fce55b8eb4bb#diff-12e598daecade60b04e493f45c134fec9287ac7471114d95d1995b4fb651a6c8.

Yes I don't know it if should be supported or not.

Reply:
Maybe we should discuss the OpenSuse topic in PR https://github.com/ghoneycutt/puppet-module-pam/pull/243